assets/javascripts/initializers/discourse-graphviz.js.es6

```diff
@@ -32,9 +32,9 @@ export default {
         hpccWasm.graphviz.layout(graphDefinition, "svg", engine).then((svgChart) => {
           $container.html(svgChart);
         }).catch(e => {
-          // graphviz error are unhelpful so we just show a default error
+          // graphviz errors are very helpful so we just show them as is
           const $error = $(
-            "<div class='graph-error'>Error while rendering graph.</div>"
+            "<div class='graph-error'>" + e.message + "</div>"
```

Comment:
is this a XSS issue ? can you use this error message as a vector for injecting JS? Should this be escaped?